ledger/controllers/wallet.js

```diff
@@ -872,6 +963,30 @@ function claimWalletHandler (runtime) {
     const wallets = runtime.database.get('wallets', debug)
     const members = runtime.database.get('members', debug)
 
+    if (runtime.config.disable.wallets) {
+      throw boom.serverUnavailable()
+    }
+
+    if (runtime.config.forward.wallets) {
+      try {
+        await runtime.wreck.walletMigration.post(debug, `/v3/wallet/uphold/${paymentId}/claim`, {
+          useProxyP: true,
```

Comment:
why are we only setting `useProxyP` on this request?

Reply:
it was probably just copied. can remove